src/components/BattleList/BattleList.js

```diff
@@ -1,18 +1,20 @@
 import React from 'react';
-import withStyles from 'isomorphic-style-loader/lib/withStyles';
+import withStyles from 'isomorphic-style-loader/withStyles';
 import { graphql, compose } from 'react-apollo';
 import PropTypes from 'prop-types';
 import { toServerTime, sortResults } from 'utils';
-import LocalTime from '../../components/LocalTime';
-import Link from '../../components/Link';
-import Time from '../../components/Time';
-import Kuski from '../../components/Kuski';
-import { BattleType } from '../../components/Names';
+import LocalTime from '../LocalTime';
```

Comment:
If i understand the webpack config right, `'components/LocalTime'` would work as well as it uses `src/` to resolve as well, and it's a clearer pattern

Reply:
I think there is a rule somewhere that says don't use absolute paths.

Reply:
Nope and I don't think there should be. Backtracking relative imports break when moving files around. I can fix all these in next refactor if you want.